rust/ruby-rbs/build.rs

```diff
@@ -147,6 +147,29 @@ fn generate(config: &Config) -> Result<(), Box<dyn Error>> {
                         )?;
                         writeln!(file, "    }}")?;
                     }
+                    "rbs_node" => {
+                        let (function_name, field_name) = match field.name.as_str() {
+                            "type" => ("r#type", "type_"),
```

Comment:
Does this mean that the method will be created with the name `type`? If so, I'd vote for us to use `type_`.

Since `type` is a reserved keyword, invoking the method might prove a little odd, especially if we end up having to use the raw identifier syntax (`r#type()`).